manifests/deploy/openshift/olm/bundle/4.5/meteringoperator.v4.5.0.clusterserviceversion.yaml

```diff
@@ -206,7 +206,7 @@ metadata:
     categories: OpenShift Optional, Monitoring
     certified: "false"
     containerImage: quay.io/openshift/origin-metering-ansible-operator:4.5
-    createdAt: 2019-01-01T11:59:59Z
+    createdAt: "2019-01-01T11:59:59Z"
```

Comment:
What version of helm are you using? This shouldn't be adding any quotes.

https://github.com/operator-framework/operator-metering/blob/master/Documentation/dev/developer-guide.md#development-dependencies

Reply:
@timflannagan1 strange...I got around it by running the make from my fedora desktop which has an older helm verison
```
pruan@fedora-vm ~METERING_HOME (fix_catalogsource●)$ helm version
Client: &version.Version{SemVer:"v2.12.2", GitCommit:"7d2b0c73d734f6586ed222a567c5d103fed435be", GitTreeState:"clean"}

```